src/panels/ShareModelPanel/ShareModel.tsx

```diff
@@ -39,28 +124,232 @@ export default function ShareModel() {
             </div>
           }
         />
-        <div className="p-panel__content">
-          <div>
+        {errorMsg && (
```

Comment:
We don't have a design for this. How can we make it fail to qa?

Reply:
This is a temporary solution using Vanilla notifications until we have the toast noifications. To make it error, try to share a username that you're already sharing with.